datahub-web-react/src/app/glossary/GlossarySearch.tsx

```diff
@@ -28,6 +29,11 @@ const ResultsWrapper = styled.div`
     top: 45px;
 `;
 
+const ParentViewWrapper = styled.div`
+    margin-bottom: 5px;
+    font-size: 8px;
```

Comment:
can remove this line of css since we need to pass in the font size to get our desired outcome

Reply:
Fixed and committed!!